src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-sound_lib.py

```diff
@@ -1,19 +1,8 @@
-# ------------------------------------------------------------------
-# Copyright (c) 2020 PyInstaller Development Team.
-#
-# This file is distributed under the terms of the GNU General Public
-# License (version 2.0 or later).
-#
-# The full license is available in LICENSE.GPL.txt, distributed with
-# this software.
-#
-# SPDX-License-Identifier: GPL-2.0-or-later
-# ------------------------------------------------------------------
-
 """
 sound_lib: http://hg.q-continuum.net/sound_lib
 """
 
 from PyInstaller.utils.hooks import collect_dynamic_libs
 
-binaries = collect_dynamic_libs('sound_lib')
+binaries = []
+datas = collect_dynamic_libs('sound_lib')
```

Comment:
Likewise, same issues and questions as the last file?

Reply:
On Mac, since the folder also contains dll files, if they are added on binaries (with binaries =), something goes wrong and it doesn't get added. But with "datas" they do.

Reply:
Ahh, just tried it and I see what you mean.